spec/integration/data_stream/fields/fields.spec.yml

```diff
@@ -355,6 +367,46 @@ spec:
         type: boolean
         default: false
 
+      # Conditional properties.
+      default_metric: true
+      metrics: true
+
+    allOf:
+      - if:
+          properties:
+            type:
+              const: aggregate_metric_double
+          required:
+            - type
+        then:
+          required:
+            - default_metric
+          properties:
+            default_metric:
```

Comment:
`type: string` is not needed here?

Reply:
`type: string` is defined in the referenced definition.